.gitignore

```diff
@@ -1,3 +1,6 @@
 target/
 .floo
 .flooignore
+*.sublime-project
+*.sublime-workspace
+*.swp
```

Comment:
I don't care _that_ much, but you could be using https://help.github.com/articles/ignoring-files/#create-a-global-gitignore rather than dumping ignore entries for your personal tools in every project.